scripts/generator/graphql_generator/csharp/Editor/ShopifyOnboardingPanel.cs.erb

```diff
@@ -47,49 +43,68 @@ namespace <%= namespace %>.SDK {
-                BottomRightAnchoredRect(Styles.BackgroundIllustration.width, Styles.BackgroundIllustration.height),
-                Styles.BackgroundIllustration
-            );
-
             GUILayout.BeginArea(
                 new Rect(
-                    Margin,
-                    Margin + AccentBarHeight,
-                    Mathf.Min(position.width - Margin * 2, DesiredWindowWidth),
-                    position.height - (Margin * 2) - AccentBarHeight
+                    0,
+                    AccentBarHeight,
+                    Mathf.Min(position.width, DesiredWindowWidth),
+                    position.height - AccentBarHeight
                 )
             );
 
-            GUILayout.Label(Styles.LogoImage, GUILayout.Height(Styles.LogoImage.height));
-            GUILayout.Label("Unity Buy SDK", Styles.HeadingStyle);
+            GUILayout.BeginVertical();
+
+            GUILayout.Space(20f);
+
+            GUILayout.Label(Styles.Illustration, Styles.HeadingStyle, GUILayout.Height(Styles.Illustration.height/3f));
+
+            GUILayout.Label("Boost your bottom line with an in-game store", Styles.HeadingStyle);
 
             GUILayout.Label(
-                "Reward your most dedicated fans with physical merchandise. Shopify Buy SDK enables players to order merchandise directly from your game experience.",
-                Styles.BodyStyle
+                "Start selling merchandise directly inside your game with the Shopify In-Game Buy SDK.",
+                Styles.AccentBody
             );
 
-            GUILayout.Label("Start your store today", Styles.SubheadingStyle);
+            GUILayout.EndVertical();
+
+            var bottomArea = EditorGUILayout.BeginVertical();
+
+            DrawRect(new Rect(0, bottomArea.yMin, position.width, bottomArea.height), Styles.AltBackgroundColor);
+            DrawRect(new Rect(0, bottomArea.yMin, position.width, 1), Styles.SeparatorColor);
+
+            GUILayout.Space(40f);
+
+            GUILayout.Label("Start your 30-day free trial", Styles.SubheadingStyle);
 
             GUILayout.Label(
-                "To use the Shopify Buy SDK you’ll need to set up a Shopify store.",
+                "In order to use this plugin, you need to sign up for a Shopify account. Your Shopify account allows you to build products, manage orders, and accept payments.",
                 Styles.BodyStyle
             );
 
-            if (GUILayout.Button("Start your free trial", Styles.ButtonStyle)) {
-                Application.OpenURL("https://www.shopify.com/signup?signup_types=unity_buy_sdk,<%= publish_destination %>");
-            }
-            
-            GUILayout.Label("Learn how to connect your game to your Shopify store", Styles.SubheadingStyle);
+            GUILayout.BeginHorizontal();
+            GUILayout.FlexibleSpace();
+            if (GUILayout.Button("Start your free trial", Styles.ButtonStyle))
+                Application.OpenURL("www.shopify.com/unity");
```

Comment:
@coreypollock this url https://www.shopify.com/unity is currently 404ing. We should also be passing in the publish destination here so that we can track where signups are coming from: `github` or `asset-store`

Reply:
@keyfer this might be of interest to you also. We're distributing code + a page in Unity that will send people to https://www.shopify.com/unity we need to be able to track that the signup came from the SDK and not organically landing at https://www.shopify.com/unity. Also it's complicated cause we'll need to track that the signup came from either the the Unity SDK being distributed via Github or the Asset Store.

This is why previously we were generarting the signup link as:
`"https://www.shopify.com/signup?signup_types=unity_buy_sdk,<%= publish_destination %>"`

Reply:
@mikkoh `/unity` will 404 until https://github.com/Shopify/brochure2/pull/8934 is shipped.

Reply:
@keyfer can we provide any info to the brochure page that will help us determine the source (github or asset store build)?

Reply:
I've added `unity_buy_sdk` as a signuptype, but there might be some more we could do.

@brysongilbert what do you think?

Reply:
@JakeCataford can we do this?
`Application.OpenURL("www.shopify.com/unity?signup_types=unity_buy_sdk,<%= publish_destination %>-unityeditor");`

So the possible url values would be:
www.shopify.com/unity?signup_types=unity_buy_sdk,github-unityeditor
www.shopify.com/unity?signup_types=unity_buy_sdk,asset-store-unityeditor

@coreypollock was also suggesting that we should track signups via the Asset Store description this should be something like this:
www.shopify.com/unity?signup_types=unity_buy_sdk,asset-store-description

@brysongilbert and @keyfer thoughts?

Reply:
It might be best to move this into a discussion in the unity channel, and invite @brysongilbert there as this comment is collapsed, and might be hard to track?